### ISSUE TITLE
It seems "weighted_sse" is not implemented 

### ISSUE THREAD

**Issue by hsnkhaki** [che-shr-cat/currennt#2]
Hi,

I try "weighted_sse" with the proper configuration (defined in README, a copy is at the end) but it returns unknown layer type:

` ./run.sh
Started in hybrid online/batch training mode.
Mini-batches (50 sequences each) will be shuffled during training.
Using input noise with a standard deviation of 0.1.
The trained network will be written to 'trained_network.jsn'.
Validation error will be calculated every 1 epochs.
Training will be stopped after 100 epochs or if there is no new lowest validatio                n error within 20 epochs.
Utilizing the GPU for computations with 50 sequences in parallel.
Normal distribution with mean=0 and sigma=0.1. Random seed: 2659768064

Using device #0 (Tesla K80)
Reading network from 'network.jsn'... done.

Loading training set 'train_1_speaker.nc' ...
using cache file: /tmp/2df4-2bb1-1592-3678
... done.
Loaded fraction:  100%
Sequences:        500
Sequence lengths: 113..216
Total timesteps:  74334

Loading validation set 'val_1_speaker.nc' ...
using cache file: /tmp/69cf-4c42-5905-2550
... done.
Loaded fraction:  100%
Sequences:        102
Sequence lengths: 113..152
Total timesteps:  13878

Creating the neural network... FAILED: Invalid network file: Could not create la                yer: Unknown layer type 'weighted_sse'
`
I am using the updated CURRENNT, currennt-0.2-rc1, which is said weighted_sse is implemented but there is no!!

PS.
From README:
"weighted_sse: Weighted Sum of Squared Error objective function
  To be used with any output layer. Requires 2N targets for output layer of
  size N. The targets and the weights are expected to be interleaved, i.e., any
  vector of targets should be of the form (t1, w1, t2, w2, ..., tN, wN)."
